modules/openapi-generator/src/main/resources/typescript/index.mustache

```diff
@@ -16,7 +16,7 @@ export { Middleware } from './middleware{{importFileExtension}}';
 export{{#platforms}}{{#deno}} type{{/deno}}{{/platforms}} { PromiseMiddleware as Middleware } from './middleware{{importFileExtension}}';
 {{/useRxJS}}
 {{#useObjectParameters}}
-export { {{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}{{classname}}{{operationIdCamelCase}}Request, {{/operation}}Object{{classname}} as {{classname}}{{^-last}}, {{/-last}} {{/operations}}{{/apis}}{{/apiInfo}}} from './types/ObjectParamAPI{{importFileExtension}}';
+export { {{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}{{#platforms}}{{#deno}}type {{/deno}}{{/platforms}}{{classname}}{{operationIdCamelCase}}Request, {{/operation}}Object{{classname}} as {{classname}}{{^-last}}, {{/-last}} {{/operations}}{{/apis}}{{/apiInfo}}} from './types/ObjectParamAPI{{importFileExtension}}';
```

Comment:
this is the meaningful change of this PR.

It makes sure that any re-exported request object is exported with the `type` keyword, e.g. 
```ts
export { AuthApiXRequest, ObjectAuthApi as AuthApi } from './types/ObjectParamAPI.ts';
```
becomes
```ts
export { type AuthApiXRequest, ObjectAuthApi as AuthApi } from './types/ObjectParamAPI.ts';
```

e.g. meaning
```diff
export {
-  AuthApiXRequest,
+  type AuthApiXRequest,
   ObjectAuthApi as AuthApi
} from './types/ObjectParamAPI.ts';
```